tests/messengers/telegram/test_tutorials.py

```diff
@@ -27,7 +27,9 @@
         "3_buttons_with_callback",
     ],
 )
-def test_client_tutorials_without_telegram(tutorial_module_name):
+@pytest.mark.telegram
```

Comment:
`telegram` mark is supposed to be used only for tests that require telegram client API (`session_file` fixture):

> telegram: marks tests as requiring telegram client API token to work